src/expr/core/src/expr/build.rs

```diff
@@ -34,7 +34,8 @@ use crate::{bail, ExprError, Result};
 
 /// Build an expression from protobuf.
 pub fn build_from_prost(prost: &ExprNode) -> Result<BoxedExpression> {
-    ExprBuilder::new_strict().build(prost)
+    let expr = ExprBuilder::new_strict().build(prost)?;
+    Ok(Strict::new(expr).boxed())
```

Comment:
Can we move this step into the `ExprBuilder`?

Reply:
This wrapper is only needed at the top level. So I think no need to move it into the builder?